assets/src/edit-story/components/dialog/dialog.js

```diff
@@ -113,7 +111,6 @@ const Dialog = ({
 };
 
 Dialog.propTypes = {
-  open: PropTypes.bool,
```

Comment:
Should we still keep `isOpen` here?

Reply:
i included it in the comment at the top of the component and am just passing it through. we can include it, i thought it was a little redundant?

Reply:
Probably redundant!